sources/meta-yoe/recipes-siot/simpleiot/simpleiot_0.14.10.bb

```diff
@@ -7,11 +7,11 @@ LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
 SECTION = "apps"
 
-DEPENDS += "go-native elm-binary-native nodejs-native nodejs-oe-cache-native gyp-native python3-packaging-native curl-native ca-certificates-native"
+DEPENDS += "go-native"
```

Comment:
hmmm will it work without elm compiler ? then we can get rid of meta-elm-binary layer too